Rozetka/Mobile.cs

```diff
@@ -4,7 +4,6 @@ public class Mobile: Product
     {
         public Mobile()
         {
-
         }
         public Mobile(string name, int price, int popularity, string numberOfReviews, string productlink)
```

Comment:
I changed Popularity on Rating in all places, made price as double. NumberOfReviews I renamed on Reviews and remain as string, because actually it is not a number, it is a text like "345 reviews", "2 reviews".  In order to convert it to int - additional logic should be written to extract only numbers from string.